canvas_course_info/settings/base.py

```diff
@@ -140,6 +140,8 @@
 ICOMMONS_API_TOKEN = SECURE_SETTINGS.get('icommons_api_token')
 ICOMMONS_BASE_URL = SECURE_SETTINGS.get('icommons_base_url')
 ICOMMONS_API_PATH = '/api/course/v2/'
+ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = True
```

Comment:
If we follow the example of canvas_account_admin_tools, it should be `False` in base.py, and overridden in local.py with `True`.